src/main/java/com/mike_caron/equivalentintegrations/storage/EMCItemHandler.java

```diff
@@ -125,7 +125,7 @@ public ItemStack insertItem(int slot, @Nonnull ItemStack stack, boolean simulate
 
             knowledge = ProjectEAPI.getTransmutationProxy().getKnowledgeProviderFor(owner);
 
-            if(forbidNbt && stack.hasTagCompound())
+            if(forbidNbt && stack.hasTagCompound() && !(stack.getTagCompound().toString().length() < 4))
```

Comment:
Rather than comparing as a string, why not just check the tag size? eg, `stack.getTagCompound().getSize() == 0`

Reply:
Didn't know getSize() even existed. I'll make the fix real quick.